media/src/activity/activity-map.tsx

```diff
@@ -578,8 +566,12 @@ export const ActivityMap: React.FC = () => {
                 sizeScale: ICON_SCALE,
                 getPosition: (d) => d.location.lng_lat,
                 onClick: pickEventClickHandler,
-                getSize: ICON_SIZE,
-                getColor: ICON_COLOR,
+                getSize: (d) => {
```

Comment:
Quick note that I'd like the marker to change when I click on it, not just when event detail is displayed.